frameworks/elastic/tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import sdk_repository as repo
+
+
+@pytest.fixture(scope='session')
+def configure_universe():
```

Comment:
Loops across the stub_urls defined in `STUB_UNIVERSE_URL`:

https://github.com/mesosphere/dcos-commons/blob/599380647c3d304aff1a6e2c40f0b0f97cd5813e/testing/sdk_repository.py#L44